examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
@agramfort This is auditory (short beeps) on left and right separately. If you would like a multimodal dataset, I can ask around for something that has both visual and auditory (maybe sef also).